libs/langchain/langchain/llms/azureml_endpoint.py

```diff
@@ -116,6 +116,21 @@ def format_response_payload(self, output: bytes) -> str:
         """
 
 
+class _DefaultContentFormatter(ContentFormatterBase):
+    """Default Content handler"""
+
+    def format_request_payload(self, prompt: str, model_kwargs: Dict) -> bytes:
```

Comment:
@yangmingzhe , could you describe the scenario where this will be used? request schema depends on model family, so providing a default schema is not going to work and likely to cause confusion to the user.

Reply:
@prakharg-msft , you're right indeed. I'll take it into serious consideration, thank you so much!